src/NServiceBus.RabbitMQ/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Fody" version="3.3.3" targetFramework="net452" developmentDependency="true" />
+  <package id="Fody" version="4.2.1" targetFramework="net452" developmentDependency="true" />
```

Comment:
Why the Fody upgrade? If you were getting an error about `IntermediateDir`, just build again, and it works.

Reply:
I tried that, I even did git clean->rebuild->build->build. It didn't matter how many times, for whatever reason that did not work for me. The 4x release included a commit to address this and after upgrading, it fixed it.

Is there some reason _not_ to upgrade Fody?

Reply:
Fody alters the assembly, so it adds to the scope of changes being introduced in the patch. There have also been bugs in the past where it corrupts the PDBs in some way that makes them unusable for debugging. That's why I prefer to not upgrade Fody in hotfixes.